content_scripts/vimium_frontend.coffee

```diff
@@ -5,7 +5,7 @@
 # "domReady".
 #
 
-findModeQuery = { rawQuery: "", matchCount: 0 }
+window.findModeQuery = { rawQuery: "", matchCount: 0 }
 findModeQueryHasResults = false
 findModeAnchorNode = null
 findModeInitialRange = null
```

Comment:
As this has been extended and extended, it might be better soon to collect these together into an object, something like `findMode.query`, `findMode.hasResults`, etc.

(Perhaps leave as is, for now, to keep the diff clean.)